github/pulls_reviews.go

```diff
@@ -5,7 +5,10 @@
 
 package github
 
-import "time"
+import (
+	"fmt"
+	"time"
+)
 
 // PullRequestReview represents a review of a pull request.
 type PullRequestReview struct {
```

Comment:
https://developer.github.com/v3/pulls/reviews/#list-reviews-on-a-pull-request

Some fields of list review API response are missing.

- commit_id
- html_url
- pull_request_url

You can see similar fields in PullRequestComment
https://github.com/google/go-github/blob/7b47f69ea1f1cb428354f0a6361242e9cb2cf63b/github/pulls_comments.go#L14

And "state" seems "UPPRECASE" and can be "APPROVED", "DISMISSED", "COMMENTED".

PullRequestReview  is also used in PullRequestReviewEvent https://developer.github.com/v3/activity/events/types/#pullrequestreviewevent , so maybe we should check consistencies and give feed backs to GitHub support perhaps if the differences are not reasonable.